templates/shared/forms/form-fields.html

```diff
@@ -0,0 +1,206 @@
+<section class="p-section">
+  <form {% if isModal %}class="js-modal-form"{% endif %}
+        action="/marketo/submit"
+        method="post"
+        id="mktoForm_{{ formData.formId }}">
+    {% for fieldset in fieldsets %}
+      <div class="p-section">
+        <hr class="is-fixed-width" />
+        <fieldset class="p-fieldset-section {% if fieldset.id=='ubuntu-versions' %} js-toggle-checkbox-visibility {% elif fieldset.isRequired and fieldset.fields[0].type=='checkbox' %} js-required-checkbox {% endif %}" id="{{ fieldset.id }}" aria-labelledby="{{ fieldset.id }}">
+        <div class="row--50-50 js-formfield">
```

Comment:
```suggestion
        <div class="row--50-50 {% if not fieldset.no-js %}js-formfield{% endif%}">
```
this is what I am talking about in [this comment](https://github.com/canonical/ubuntu.com/pull/14215#discussion_r1742497272), fyi, this is psudo code